### PR TITLE
chore: update deps, remove deprecated gulp-util

### LINKED ISSUE #14: gulp-util have been deprecated

### PR DESCRIPTION
**Pull request by avdeev**
Fix #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,9 @@
     "gulp"
   ],
   "dependencies": {
-    "gulp-util": "~2.2.14",
-    "through2": "~0.6.3"
+    "plugin-error": "0.1.2",
+    "through2": "2.0.3",
+    "vinyl": "2.1.0"
```
Comment (demurgos) [inline package.json]: `vinyl`  should be in the `devDependencies`

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,377 @@
+{
```
Comment (demurgos) [inline package-lock.json]: Using lock files for libraries is not an obvious choice. Please keep this PR only about dropping `gulp-util`. Unless you know the opinion of the maintainer about it, please open a second PR if you want to discuss the addition of the lock file.

File: package.json (hunk 1 above)
Comment (mariusGundersen) [reply package.json]: How can vinyl be in devDependencies when it's used by the code? Did you mean that it should be in peer-dependencies?

Comment (demurgos) [reply package.json]: I only see it used in `test.js` but I may be missing something.

Comment (mariusGundersen) [reply package.json]: Yup, my bad, you're right

**Comment by avdeev** [comment]
Thank you! I will fix it.

File: package.json (hunk 1 above)
Comment (demurgos) [reply package.json]: Could you fix it and push the change?

Comment (avdeev) [reply package.json]: Thank you! I will fix it.

**Comment by avdeev** [comment]
@demurgos @mariusGundersen Remarks in the review have been fixed
Review again, please

**Pull request merged**